pkg/cloud/cloud.go

```diff
@@ -41,11 +41,13 @@ const (
 	// PollCheckInterval specifies the interval to check if filesystem is ready;
 	// needs to be shorter than the provisioner timeout
 	PollCheckInterval = 30 * time.Second
-	// PollCheckTimeout specifies the time limit for polling DescribeFileSystems
-	// for a completed create/update operation. FSx for Lustre filesystem
-	// creation time is around 5 minutes, and update time varies depending on
+	// PollCheckTimeout specifies the time limit for polling
+	// DescribeFileSystems/DescribeDataRepositoryAssociation for a completed
+	// create/update operation. FSx for Lustre filesystem
+	// creation time is around 5 minutes, Data Repository Association
+	// creation time is around 10 minutes, and update time varies depending on
 	// target file system values
-	PollCheckTimeout = 10 * time.Minute
+	PollCheckTimeout = 15 * time.Minute
```

Comment:
If PollCheckTimeout < provisionerTimeout, the provisionerTimeout will always kill the CreateVolume call before it the PollCheckTimeout is hit. I don't think incrementing this should make a difference